gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/account/AccountEndpointHandler.java

```diff
@@ -0,0 +1,151 @@
+import io.gravitee.am.service.FactorService;
+import io.reactivex.Observable;
+import io.reactivex.Single;
+import io.vertx.reactivex.ext.web.RoutingContext;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class AccountEndpointHandler {
+    private static final int DEFAULT_PAGE_SIZE = 20;
+    private static final int DEFAULT_PAGE = 0;
+    private final UserService userService;
+    private final FactorService factorService;
+    private final ActivityAuditService activityAuditService;
+    private final ClientSyncService clientSyncService;
+    private final Domain domain;
+
+    public AccountEndpointHandler(UserService userService,
+                                  FactorService factorService,
+                                  ActivityAuditService activityAuditService,
+                                  ClientSyncService clientSyncService,
+                                  Domain domain) {
+        this.userService = userService;
+        this.factorService = factorService;
+        this.activityAuditService = activityAuditService;
+        this.clientSyncService = clientSyncService;
+        this.domain = domain;
+    }
+
+    public void getUser(RoutingContext routingContext) {
+        JWT token = routingContext.get(ConstantKeys.TOKEN_CONTEXT_KEY);
+        userService.findById(token.getSub()).doOnError(err -> {
+            //todo: handle error
+        }).toSingle().subscribe(user -> {
+            routingContext.put(ConstantKeys.USER_CONTEXT_KEY, user);
+            routingContext.next();
+        });
+    }
+
+    //mounted angular
+    public void getAccount(RoutingContext routingContext) {
+        User user = routingContext.get(ConstantKeys.USER_CONTEXT_KEY);
+        AccountResponseHandler.handleWIP(routingContext);
+    }
+
+    //static assets
+    public void getAsset(RoutingContext routingContext) {
+        User user = routingContext.get(ConstantKeys.USER_CONTEXT_KEY);
+        AccountResponseHandler.handleWIP(routingContext);
+    }
+
+    public void updateDevices(RoutingContext routingContext) {
+        //TODO: waiting to impl fully
+        User user = routingContext.get(ConstantKeys.USER_CONTEXT_KEY);
+        AccountResponseHandler.handleWIP(routingContext);
+    }
+
+    public void getProfile(RoutingContext routingContext) {
+        AccountResponseHandler.handleDefaultResponse(routingContext, routingContext.get(ConstantKeys.USER_CONTEXT_KEY));
+    }
+
+    public void getActivity(RoutingContext routingContext) {
+        getActivityAudit(routingContext, routingContext.get(ConstantKeys.USER_CONTEXT_KEY)).subscribe(result -> AccountResponseHandler.handleDefaultResponse(routingContext, result));
+    }
+
+    public void getDevices(RoutingContext routingContext) {
+        collectFactors(routingContext.get(ConstantKeys.USER_CONTEXT_KEY)).subscribe(factors -> AccountResponseHandler.handleDefaultResponse(routingContext, factors));
```

Comment:
This endpoint return the Factor plugin configuration : 
```
[ {
  "id" : "9bc62d13-c61d-4770-862d-13c61db7702d",
  "name" : "SMS Factor",
  "type" : "sms-am-factor",
  "factorType" : "SMS",
  "configuration" : "{\"countryCodes\":\"fr\",\"graviteeResource\":\"2708c49f-26c7-433f-88c4-9f26c7933fd2\"}",
  "domain" : "5e916548-a785-4efa-9165-48a7858efa7d",
  "createdAt" : 1625125542643,
  "updatedAt" : 1625125542643
} ]
```
According to the name "devices" I'm wondering if the output should be the "factors" entry into the User object : 
```
"factors" : [ {
    "factorId" : "9bc62d13-c61d-4770-862d-13c61db7702d",
    "channel" : {
      "type" : "SMS",
      "target" : "+33645152508"
    },
    "createdAt" : 1625125736595,
    "updatedAt" : 1625125736595
  } ],
``` 
Maybe I'm wrong, we should check this point with our Product Owner